### DISCUSSION TITLE
EddieQuiz or EddieHubQuiz?

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#1196] [Ideas]
Can anyone create **EddieQuiz** or **EddieHubQuiz**? Something like https://quizizz.com/ or any related about that.

`Features:`
- Timer
- Multiple choice
- Leaderboard

~~-~~

`Questions:`
- When did Eddie Jaoude created this organization/community
- When did the organization rename into EddieHub?
- Who is the first moderator?
- What is Open Source?
- Etc.

We should have many questions.

**Comment by eddiejaoude** [comment]
Oh cool 👍
